tests/acceptance/features/apiSpaces/copySpaces.feature

```diff
@@ -451,3 +451,248 @@ Feature: copy file
       | permissions |
       | 31          |
       | 17          |
+
+
+  Scenario: Copying a file to a folder with no permissions
+    Given using spaces DAV path
```

Comment:
Isn't that the case for every single test in local api tests?

Reply:
There are many scenarios up that does not require this step.